e2e/portfolio.spec.ts

```diff
@@ -0,0 +1,143 @@
+      'URL must link directly to an image file'
+    );
+    await page
+      .getByLabel(translations.settings.labels.image)
+      .fill('http://helloworld.com/image.png');
+    await expect(page.getByTestId('image-validation')).toBeHidden();
+  });
+
+  test('The description has validation', async ({ page }) => {
+    await page
+      .getByRole('button', { name: 'Add a new portfolio Item' })
+      .click();
+    await page
+      .getByLabel(translations.settings.labels.description)
+      .fill(
+        'This is the longest description you will ever see in your entire life, you will never see such a long description again. This is the longest description in existence. Nothing will ever come close to be being so long again in the entire history of the world. I only have 30 characters left.'
+      );
+    await expect(page.getByTestId('description-validation')).toContainText(
+      'There is a maximum limit of 288 characters, you have 0 left'
+    );
+    await page
+      .getByLabel(translations.settings.labels.description)
+      .fill('My description');
+    await expect(page.getByTestId('description-validation')).toBeHidden();
+  });
+
+  test('It should be possible to delete a portfolio item', async ({ page }) => {
+    await page
+      .getByRole('button', { name: 'Add a new portfolio Item' })
+      .click();
+    await page
+      .getByLabel(translations.settings.labels.title)
+      .fill('My portfolio');
+    await page
+      .getByLabel(translations.settings.labels.url)
+      .fill('https://my-portfolio.com');
+    await page
+      .getByLabel(translations.settings.labels.image)
+      .fill('https://my-portfolio.com/image.png');
+    await page
+      .getByLabel(translations.settings.labels.description)
+      .fill('My description');
+
+    await page
+      .getByRole('button', { name: 'Remove this portfolio item' })
+      .click();
+
+    await expect(page.getByTestId('portfolio-items')).toBeHidden();
+  });
+
+  test('It should be possible to add a portfolio item', async ({ page }) => {
+    await page
+      .getByRole('button', { name: 'Add a new portfolio Item' })
+      .click();
+    await expect(
+      page.getByRole('button', { name: 'Add a new portfolio Item' })
+    ).toBeDisabled();
+
+    await page
+      .getByLabel(translations.settings.labels.title)
+      .fill('My portfolio');
+    await page
+      .getByLabel(translations.settings.labels.url)
+      .fill('https://my-portfolio.com');
+    await page
+      .getByLabel(translations.settings.labels.image)
+      .fill('https://my-portfolio.com/image.png');
+    await page
+      .getByLabel(translations.settings.labels.description)
+      .fill('My description');
+
+    await page
+      .getByRole('button', { name: 'Save this portfolio item' })
+      .click();
+    await expect(page.getByTestId('flash-message')).toContainText(
+      /We have updated your portfolio/
+    );
+  });
+
+  // TODO: Add test for viewing portfolio item
```

Comment:
is this still a todo item?
or can this comment be deleted?

Reply:
It is something I want to add after the migration